staging/src/k8s.io/apiserver/pkg/audit/policy/reader_test.go

```diff
@@ -70,6 +71,18 @@ rules:
   - level: Metadata
 `
 
+const policyWithUnknownField = `
+apiVersion: audit.k8s.io/v1
+kind: Policy
+rules:
+- level: None
+  resources:
+  - group: coordination.k8s.io
+    resources:
+    - "leases"
+    verbs: ["watch", "get", "list"]
```

Comment:
nit: add a comment to indicate that this is the known field
```suggestion
    verbs: ["watch", "get", "list"]  # invalid indentation on verbs
```